src/pages/guides/getting_started/code_playground_workflow.md

```diff
@@ -0,0 +1,193 @@
+| **Increase font size**           | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Plus (+)</kbd>     | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Plus (+)</kbd>     |
+| **Decrease font size**           | <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>Minus (-)</kbd>    | <kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>Minus (-)</kbd>    |
+| **Switch between tabs**          | <kbd>Ctrl</kbd> + <kbd>1, 2, 3, 4</kbd>                      | <kbd>Cmd</kbd> + <kbd>1, 2, 3, 4</kbd>                      |
+| **View the typings suggestions** | <kbd>Ctrl</kbd> + <kbd>space</kbd>                           | <kbd>Cmd</kbd> + <kbd>space</kbd>                           |
+
+<InlineAlert slots="header,text1" variant="success"/>
+
+#### TIP
+
+Use the "**...**" button in the top right corner of the playground window to reference the available keyboard shortcuts, start a new session, link to documentation and more.
+
+## Session Management
+
+### Saving Your Work
+
+The Code Playground features local persistence to help prevent the loss of your work. This functionality ensures that your code is stored in your browser's local storage, providing a safeguard against accidental data loss.
+
+Code in the playground is **_not saved automatically_**. To ensure it's saved, you need to take one of the following steps:
+
+1. Save your work using the [keyboard shortcut for Save](#keyboard-shortcuts)
+2. Run the code via the **Run Code** button or with the [keyboard shortcut for Run](#keyboard-shortcuts)
+3. Exit the playground (with the **X** in the upper right corner)
+
+If you don't want to save your work at any time, use the [keyboard shortcut to Reset](#keyboard-shortcuts).
+
+<InlineAlert slots="header, text1" variant="warning"/>
+
+#### IMPORTANT
+
+- Only your most recent session is saved
+- Storage is browser-specific (not synced across devices)
+- Code is not saved in incognito/private browsing modes
+- Clearing browser data will delete saved code
+
+### Resuming Sessions
+
+There are two ways to resume working on your last saved session:
+
+#### Via the Add-ons Panel
+
+1. With any document open, click the **Add-ons** button in the left rail
+2. Select the **Your add-ons** tab
+3. Toggle on **Code Playground** at the bottom of the panel
+
+![Code Playground Add-on Mode](./img/playground-on.png)
+
+<InlineAlert slots="text1" variant="info"/>
+
+If you haven't enabled Add-on Development mode yet, you'll be prompted to do so. See [Enable Add-on Development Mode](./hello-world.md#enable-add-on-development-mode) for detailed instructions.
+
+#### Via the Your add-ons Page
+
+1. The **Your add-ons** page where you manage your add-ons now features a dedicated section for the playground, allowing you to quickly access your last session or create a new one
+2. Find the **Playground Sessions** section in the **Your add-ons** page
```

Comment:
the UI has undergone changes. Replace 'Playground Sessions section' with 'Playground Sessions tab'